apps/src/gamelab/PropTypes.js

```diff
@@ -108,6 +111,40 @@ function getSerializedAnimationProps(animation) {
     'version'
   ]);
 }
+
+/**
+ * @param {!SerializedAnimationList} serializedAnimationList
+ * @throws {Error} if the list is not in a valid format.
+ */
+export function throwIfSerializedAnimationListIsInvalid(serializedAnimationList) {
```

Comment:
I'd like to reuse this simple validator on the actual level edit page, to give immediate feedback if the JSON is not a valid `SerializedAnimationList`.